New hosts UI features (#1736)
theforeman/foreman-documentation: guides/common/modules/proc_removing-packages-from-a-host.adoc

> 
> [id="removing-packages-from-a-host_{context}"]
> = Removing packages from a Host

Reviewer: ```suggestion
[id="removing-packages-from-a-host_{context}"]
= Removing Packages from a Host
```

You can remove packages from a host in the {ProjectWebUI}.

.Procedure
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
. Click the name of the host you want to modify.
. Click the *Content* tab, then click the *Packages* tab.

> . Click the vertical ellipsis icon for the package you want to remove, then click *Remove*.

Reviewer: ```suggestion
. Click the vertical ellipsis for the package you want to remove and select *Remove*.
```
